bindings/python/pinocchio/visualize/meshcat_visualizer.py

```diff
@@ -589,6 +600,39 @@ def displayVisuals(self, visibility):
         if visibility:
             self.updatePlacements(pin.GeometryType.VISUAL)
 
+    def drawFrame(
+        self, frame_id, axis_length=0.2, axis_width=2
+    ):
+        import meshcat.geometry as mg
+        frame_name = self.model.frames[frame_id].name
+        frame_viz_name = f"{self.viewerFramesGroupName}/{frame_name}"
+        self.viewer[frame_viz_name].set_object(
+            mg.LineSegments(
+                mg.PointsGeometry(
+                    position=axis_length * FRAME_AXIS_POSITIONS,
+                    color=FRAME_AXIS_COLORS,
+                ),
+                mg.LineBasicMaterial(
+                    linewidth=axis_width,
+                    vertexColors=True,
+                ),
+            )
+        )
```

Comment:
Do we really need to set those objects each and every time we draw a frame ?

Reply:
Definitely not. We could initialize them once and only update their transforms in a loop.

As long as we expose a mechanism to update the set of objects in case frames are added/removed.

Reply:
@nim65s I just pushed an alternate implementation that does something like what you said. Updated the syntax in the PR description.